invenio_circulation/views.py

```diff
@@ -47,8 +47,8 @@ def build_url_action_for_pid(pid, action):
 
 def _get_loan_endpoint_options(app):
     """Return the configured endpoint options."""
-    endpoints = app.config.get("CIRCULATION_REST_ENDPOINTS", [])
-    options = deepcopy(endpoints.get(CIRCULATION_LOAN_PID_TYPE, {}))
+    endpoints = app.config["RECORDS_REST_ENDPOINTS"]
```

Comment:
For reviewers, this is the only real change. It is needed because otherwise the overridden config in the Invenio app was not taken into account.